### ISSUE TITLE
Allow edits to show up as new events if wanted

### ISSUE THREAD

**Issue by MTRNord** [vector-im/element-web#17317]
#### Is your suggestion related to a problem? Please describe.
In the TWIM room often edits happen days later causing me to only see them due to the hidden events labs feature. 

#### Describe the solution you'd like.
A button or option to make them show up as regular messages in the timeline with some indication that they are edits would make finding the edited event way easier and makes it easier to see something changed. Though most likely it shouldnt happen if the original message happened just a few minutes ago and is still in view.

#### Describe alternatives you've considered.
Didn't come up with alternatives. Not sure if a "jump to edit" button on the hidden event would work maybe.

**Comment by t3chguy** [comment]
This would probably do better as a spec suggestion open for any client to implement as a Proof of Concept

**Comment by MTRNord** [comment]
> 
> 
> This would probably do better as a spec suggestion open for any client to implement as a Proof of Concept
> 
> Edit:
> Oh do you mean a recipient-end option for edits to show as new messages rather than a sender-end option?

Yes it is meant on recipient-end :) Not the sender deciding if it shows up new. As in "I as the receiving person want to see that a message got edited and how it now looks like" Instead of either not knowing at all or staring at the raw event :)
